core/modules/wiki.js

```diff
@@ -522,7 +522,7 @@ exports.getTagMap = function() {
 /*
 Lookup a given tiddler and return a list of all the tiddlers that include it in the specified list field
 */
-exports.findListingsOfTiddler = function(targetTitle,fieldName) {
```

Comment:
Here you've deleted the original `findListingsOfTiddler()` method. That breaks backwards compatibilty. I was suggesting instead that the `findListingsOfTiddler()` be refactored to use the new `findTiddlersByField()` method.

Reply:
I'm still confused about this: `findTiddlersByField()` is not exists anywhere. Earlier (https://github.com/Jermolene/TiddlyWiki5/pull/3511#issuecomment-437330680) you said, that I need to implement. So you would like me to create this method and replace the contents of `findListingsOfTiddler()` with the calling of the new method (so the it would just be an "alias")?

Reply:
In other words: would you like me to extract the body of `findListingsOfTiddler()` as a new `findTiddlersByField()` method, then call the latter from it?